src/client/core/utils/dom.js

```diff
@@ -246,7 +246,7 @@ function getInvisibleElements (elements) {
 }
 
 function getTabIndexAttributeIntValue (el) {
-    let tabIndex = el.getAttribute('tabIndex');
+    let tabIndex = el.getAttribute('tabindex');
```

Comment:
I guess we should use the `nativeMethods.getAttribute` method from `hammerhead` here.